configs/sites/tier1/hera/packages.yaml

```diff
@@ -120,6 +113,10 @@ packages:
     externals:
     - spec: pkg-config@0.27.1
       prefix: /usr
+  qt:
+    externals:
+    - spec: qt@5.12.6
+      prefix: /scratch1/NCEPDEV/nems/role.epic/installs/qt-5.12.6/qtbase
```

Comment:
Is the plan to eventually install the external qt's under contrib space on various systems?

Reply:
I don't know about plans. I remember is was PITA to install it, you had to login to do that (used Dom's account). But lately I found it was easy to install from github, something like 'research' version.
Also, some systems have already installations of qt, but I couldn't use it because it was built with different gcc version and we couldn't link it with ecflow under spack-stack.